pkg/apis/workload/v1alpha1/types.go

```diff
@@ -130,6 +130,9 @@ const (
 
 	// ErrorStartingAPIImporterReason indicates an error starting the API Importer.
 	ErrorStartingAPIImporterReason = "ErrorStartingAPIImporter"
+
+	// ErrorHeartbeatMissedReason indicates that a heartbeat update was not received within the configured threshold.
+	ErrorHeartbeatMissedReason = "ErrorHeartbeat"
```

Comment:
nit: -> `ErrorHeartbeatMissed` ?

Reply:
Am confused. Where is the heartbeat stored? Do we update some condition timestamp regularly?

Reply:
Also all these are reasons, not conditions. Every condition should have a clear owner. Who owns WorkloadClusterReadyCondition ?